components/cam/src/physics/cam/micro_p3_interface.F90

```diff
@@ -623,6 +656,69 @@ subroutine micro_p3_init(pbuf2d)
       end if
    end if
 
+   if (do_prescribed_CCN) then !intialize mon_ccn_1 and mon_ccn_2
```

Comment:
It seems like in the init step you simply repeat what you did whenever you call `get_prescribed_CCN` below.  Could you just call that routine instead of copying the same code here?

Reply:
Yeah, I like the idea of reusing get_prescribed_CCN for init. Less code to maintain.